dat/missions/neutral/pirbounty_dead.lua

```diff
@@ -227,12 +234,14 @@ function pilot_board ()
       local t = subdue_text[ rnd.rnd( 1, #subdue_text ) ]:format( name )
       tk.msg( subdue_title, t )
       succeed()
+      misn.appendLog("Pirate ship boarded")
       target_killed = false
       target_ship:changeAI( "dummy" )
       target_ship:setHilight( false )
       target_ship:disable() -- Stop it from coming back
       if death_hook ~= nil then hook.rm( death_hook ) end
    else
+      misn.appendLog("Failed to board pirate ship") 
```

Comment:
Assuming the log is supposed to be a useful reference for important past events, this is just cluttering it up with unnecessary information.